### ISSUE TITLE
crash when trying to customise LNPopupBar appearance

### ISSUE THREAD

**Issue by airbob** [LeoNatan/LNPopupController#17]
Hi @LeoNatan ,great work on LNPopupController!
I tried to uncomment the code in your example project
```
+ (void)load
{
	[[LNPopupBar appearanceWhenContainedInInstancesOfClasses:@[[UINavigationController class]]] setTitleTextAttributes:@{NSFontAttributeName: [UIFont fontWithName:@"Chalkduster" size:14], NSForegroundColorAttributeName: [UIColor yellowColor]}];
	[[LNPopupBar appearanceWhenContainedInInstancesOfClasses:@[[UINavigationController class]]] setSubtitleTextAttributes:@{NSFontAttributeName: [UIFont fontWithName:@"Chalkduster" size:12], NSForegroundColorAttributeName: [UIColor greenColor]}];
	[[LNPopupBar appearanceWhenContainedInInstancesOfClasses:@[[UINavigationController class]]] setBarStyle:UIBarStyleBlack];
	[[LNPopupBar appearanceWhenContainedInInstancesOfClasses:@[[UINavigationController class]]] setTintColor:[UIColor yellowColor]];
}
```
 to see customised appearance of LNPopupBar, it was crashed with ```unrecognized selector sent to class```, am I missing anything?

**Comment by LeoNatan** [comment]
Hmm strange issue. What OS? Thanks for the report.

**Comment by airbob** [comment]
@LeoNatan your reply reminded me that I have tried on iOS8, yes, I just checked that 
```
+ (instancetype)appearanceWhenContainedInInstancesOfClasses:(NSArray<Class <UIAppearanceContainer>> *)containerTypes NS_AVAILABLE_IOS(9_0);
``` 
which is only available for 9.0 

So is there any chance apply customised color for iOS8 version(let's say for instance I want a pop up bar tint different with tab bar tint color, without changing the LNPopupController source code)?

**Comment by LeoNatan** [comment]
Yes, of course. Instead of using `appearanceWhenContainedInInstancesOfClasses`, use the old `appearanceWhenContainedIn` methods. I will update the demo project for support in iOS 8.
